mesonbuild/programs.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, name: str, command: T.Optional[T.List[str]] = None,
         self.path: T.Optional[str] = None
         self.cached_version: T.Optional[str] = None
         if command is not None:
-            self.command = mesonlib.listify(command)
+            self.command: T.List[str] = mesonlib.listify(command)
```

Comment:
This shouldn't need an annotation, mypy should know that command `List[str]` at this point